modules/serverless-functions-secrets-configmaps-interactively-specialized.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively-specialized_{context}"]
+= Modifying function access to secrets and config maps interactively with specialized commands
+
+Every time you run the `kn func config` utility, you need to navigate the entire dialogue to select the operation you need. To save steps, you can directly execute a specific operation by running a more specific form of the `kn func config` command:
+
+* To list configured environment variables:
++
+[source,terminal]
+----
+$ kn func config envs [-p <path>]
```

Comment:
What is `<path>` throughout this topic. Or, will users know what it is?

Reply:
I've done it a bit differently: s/\<path\>/\<function-project-path\>/